.circleci/create_circleci_config.py

```diff
@@ -40,9 +40,22 @@ class EmptyJob:
     job_name = "empty"
 
     def to_dict(self):
+        steps = [{"run": 'ls -la'}]
+        if self.job_name == "collection_job":
+            steps.extend(
+                [
+                    "checkout",
+                    {"run": "pip install requests"},
+                    {"run": """while [[ $(curl --location --request GET "https://circleci.com/api/v2/workflow/$CIRCLE_WORKFLOW_ID/job" --header "Circle-Token: $CCI_TOKEN"| jq -r '.items[]|select(.name != "collection_job")|.status' | grep -c "running") -gt 0 ]]; do sleep 5; done"""},
+                    {"run": 'python utils/process_circleci_workflow_test_reports.py --workflow_id $CIRCLE_WORKFLOW_ID'},
```

Comment:
In circle-ci can't we just use something like `with` or require? It will automatically wait for the `run_test` to run, and then fetch results? 
(It could even be a github action no?)

Reply:
Job 2 `require` Job 1: if job 1 failed, job 2 won't be triggered (sad 😢 ). That is why such wait is implemented (although I don't like it).

Reply:
OK then you can merge but I would like for this test to not block merge / not be ever red! let's add that it always returns true maybe?

Reply:
You mean if this new job fails, it shouldn't block the merge?

I will see if I can configure this 🤔